huaweicloud/services/eip/resource_huaweicloud_vpc_bandwidth.go

```diff
@@ -67,10 +68,32 @@ func ResourceVpcBandWidthV2() *schema.Resource {
 				ForceNew: true,
 				Computed: true,
 			},
-			"charging_mode": common.SchemaChargingMode(nil),
-			"period_unit":   common.SchemaPeriodUnit(nil),
-			"period":        common.SchemaPeriod(nil),
-			"auto_renew":    common.SchemaAutoRenewUpdatable(nil),
+
+			// charge info: period_unit, period make ForceNew false here but do nothing in update method,
+			// and charging_mode only support changing post-paid to pre-paid.
```

Comment:
it's better to update the comment
```suggestion
			// charging_mode,  period_unit and period only support changing post-paid to pre-paid billing mode.
```